var/spack/repos/builtin/packages/py-bluepysnap/package.py

```diff
@@ -14,40 +14,16 @@ class PyBluepysnap(PythonPackage):
     url      = "https://pypi.io/packages/source/b/bluepysnap/bluepysnap-0.12.0.tar.gz"
 
     version('0.12.0', sha256='a2cc24031905310941296ca975814ee0c0f27229a2c133d6a13ca92cc8dc6c9b')
-    version('0.11.0', sha256='01360cd20d04be9d0458bf8eda154299857a5b3c72db8babedd336c7740d51b9')
-    version('0.10.0', sha256='0cc73e82c5f54a2eb6e4b22b59bc1c714915bd8df7fec156af5f2e24755e805e')
-    version('0.9.0', sha256='140e06b9a7cc90719ffaf4c71ffaa9320c13bdd8ef25ffb598fd348b850b6695')
-    version('0.8.0', sha256='10337610cfb83121f2869cec53830de04eed8e90b2b27aba36b8799468fa9c0f')
-    version('0.7.0', sha256='fa4d54539fdb98c5febdabf7f13786567fc8fbd5e86f95864b4de89f18fd97bd')
-    version('0.6.1', sha256='f17cdd43a9f444e4825ab9578b3184fb17df76b1598852d3b74161647a096285')
-    version('0.5.2', sha256='d97c3fcd05261ba68e855e828b88d37fa1a7302abd79163d2d8ee806b1aad9b3')
-    version('0.5.1', sha256='81cbeab26b219b52f496ea55145e60f6b826c697265f4fe2d1effe5df249cabf')
-    version('0.5.0', sha256='97a31644d1e1daccb708e92734dfb873bc0b7aa7f98939527b91cde895cdef74')
-    version('0.4.1', sha256='cbb16a0cbd43ae4ad9e35b2fad0c986ebf9029e386087c0a934565da428ad558')
-    version('0.2.0', sha256='95273bedee0ad0b9957aed40dadc94c4a0973ba014bbc215172a9c2e7144d64a')
-    version('0.1.2', sha256='aa29c1344258c1ca0cf7f5f53f3080025b9d098e3366872369586ad7ccf334a2')
 
+    depends_on('python@3.6:', type=('build', 'run'))
```

Comment:
So we drop `python2` support. (I imagine it has been widely announced, but it's not specified in the PR description).

Reply:
We are not dropping python2 support now, it was dropped in version 0.8.0.
The old versions of `bluepysnap` defined in this file were not built in the current archives, so removing them from `package.py` has no real effect on the available modules.
With the old configuration, they could have been built manually using `spack install`, but I bet that it would fail.
For reference, the versions available in the past archives are:
```
2020-08/py-bluepysnap/0.5.2
2020-08/py-bluepysnap/0.6.1
2020-09/py-bluepysnap/0.6.1
2020-10/py-bluepysnap/0.6.1
2020-11/py-bluepysnap/0.6.1
2020-12/py-bluepysnap/0.6.1
2021-01/py-bluepysnap/0.6.1
2021-01/py-bluepysnap/0.9.0
2021-02/py-bluepysnap/0.9.0
2021-03/py-bluepysnap/0.10.0
2021-03/py-bluepysnap/0.9.0
2021-04/py-bluepysnap/0.10.0
2021-04/py-bluepysnap/0.11.0
2021-05/py-bluepysnap/0.11.0
2021-05/py-bluepysnap/0.12.0
2021-06/py-bluepysnap/0.12.0
2021-07/py-bluepysnap/0.12.0
2021-08/py-bluepysnap/0.12.0
2021-09/py-bluepysnap/0.12.0
2021-10/py-bluepysnap/0.12.0
2021-11/py-bluepysnap/0.12.0
```

Reply:
Thanks, got it.

Reply:
Thanks, got it!